server/backends/blobstore/blobstore.go

```diff
@@ -240,6 +247,8 @@ func (d *DiskBlobStore) BlobExists(ctx context.Context, blobName string) (bool,
 	if err != nil {
 		return false, err
 	}
+	ctx, spn := tracing.StartSpan(ctx)
+	defer spn.End()
 	return disk.FileExists(fullPath)
 }
```

Comment:
you tell me, but maybe we'd want to add a ctx param to disk.FileExists and just do that there?

Reply:
Makes sense; I like that.